File: Ask9/ask9.c
#include <stdio.h>

int main(){
    int payment = 5000; //Δόση
    int total = 0; //Σύνολο 
    int i = 0;
    do{
        total+=payment;
        payment*=2;
        i++;
    } while (total <= 600000);
    printf("Goal will be reached in %d weeks",i);
    if(total>600000){
        printf("Left: %d",total-600000);
    }
    return 0;
}

File: Ask9/ask9.py
payment = 5000 #Δόση
total = 0 #Σύνολο
i = 0

while total <=600000:
    total+=payment
    payment*=2
    i+=1
print('Χρειάζονται ',i,' εβδομάδες.')
if total > 600000:
    print('Υπολέίπονται ',total-600000,'δρχ.')